src/frontend/utils/initWebinarContext.spec.ts

```diff
@@ -69,6 +70,7 @@ describe('initWebinarContext', () => {
     expect(mockCheckLtiToken).not.toHaveBeenCalled();
     expect(mockGetLiveRegistrations).not.toHaveBeenCalled();
     expect(mockPushAttendance).not.toHaveBeenCalled();
+    expect(useLiveRegistration.getState().liveRegistration).toBeUndefined();
```

Comment:
How does it become undefined ?

Reply:
It doesn't, I don't why I added this... it's a mistake and failing test proves it !